test/robustness/CMakeLists.txt

```diff
@@ -15,6 +15,12 @@ function(boost_geometry_add_robustness_test item)
     PRIVATE
       Boost::geometry)
 
+  # Link to additional libraries required for testing.
+  target_link_libraries(${robustness_test_name}
+    PRIVATE
+      Boost::crc
+      Boost::program_options)
```

Comment:
This version works for me, thanks.

I will merge it today.

Reply:
Merged! Thanks again for your contribution!
It's the setup of our repo, not everyone can merge.